visualization/include/visualization/circle.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "point.hpp"
+
+namespace visualization::geometry {
+
+    class Circle {
+
+        public:
```

Comment:
the indentation is off compared to [other files](https://github.com/charlesvdv/boulot-robot/blob/master/path-planning/include/path-planning/point.hpp)